### PR TITLE
Edit contacts in group list

### PR DESCRIPTION
**Pull request by michaelkirk**
* properly populate phone fields
* Fix appearence for CNContactViewController
* "Add Contact" vs "Show Contact"

PTAL @charlesmchen

### CODE REVIEW

File: Signal/src/ViewControllers/ShowGroupMembersViewController.m [hunk 1]
```diff
@@ -280,37 +296,43 @@ - (void)showContactInfoViewForRecipientId:(NSString *)recipientId
...
+        
+        contactViewController = [CNContactViewController viewControllerForNewContact:newContact];
+    }
 
-        ABRecordRef aContact = ABPersonCreate();
-        CFErrorRef anError   = NULL;
+    contactViewController.delegate = self;
+    contactViewController.allowsActions = NO;
+    contactViewController.allowsEditing = YES;
+    contactViewController.navigationItem.leftBarButtonItem =
```
Comment (charlesmchen) [inline Signal/src/ViewControllers/ShowGroupMembersViewController.m]: We're going to let users edit contacts from other places (like 1:1 conversation settings) too, right? I wonder if we should move this to a helper class (e.g. `ViewControllerUtils`)?

File: Signal/src/ViewControllers/ShowGroupMembersViewController.m [hunk 2]
```diff
@@ -325,10 +347,20 @@ - (void)callMember:(NSString *)recipientId
     [Environment callUserWithIdentifier:recipientId];
 }
 
+- (void)dismissPressed
+{
+    DDLogDebug(@"%@ %s", self.tag, __PRETTY_FUNCTION__);
```
Comment (charlesmchen) [inline Signal/src/ViewControllers/ShowGroupMembersViewController.m]: Remove?

File: Signal/src/ViewControllers/ShowGroupMembersViewController.m [hunk 3]
```diff
@@ -325,10 +347,20 @@ - (void)callMember:(NSString *)recipientId
...
+    [self dismissViewControllerAnimated:YES completion:nil];
+}
+
 #pragma mark - ContactsViewHelperDelegate
 
 - (void)contactsViewHelperDidUpdateContacts
 {
+    // FIXME new names are not immediately chown.
+    // doing some debugging, it seems that even though [self.tableView reloadData] is called
+    // we don't see subsequent invocations of `cellForIndexpath:` unless the items are scrolled off screen.
```
Comment (charlesmchen) [inline Signal/src/ViewControllers/ShowGroupMembersViewController.m]: We can remove this now right?

Comment (michaelkirk) [inline Signal/src/ViewControllers/ShowGroupMembersViewController.m]: done

File: Signal/src/ViewControllers/ShowGroupMembersViewController.m (hunk 1 above)
Comment (michaelkirk) [inline Signal/src/ViewControllers/ShowGroupMembersViewController.m]: Done in an upcoming PR.

**Pull request merged**